backend/src/layer/data/database/request.rs

```diff
@@ -428,13 +429,13 @@ mod tests {
         assert_eq!(meals, provide_dummy_meals());
 
         let meals_in_future = request
-            .get_meals(line_id, Local::now().date_naive() + Duration::weeks(4))
+            .get_meals(line_id, Local::now().date_naive() + Duration::weeks(5))
             .await
```

Comment:
Replace the 5 with MAX_WEEKS_DATA +1 and 4 with MAX_WEEKS_DATA

Reply:
ok